pkg/microservice/aslan/config/consts.go

```diff
@@ -220,3 +223,29 @@ const (
 	ImageResourceType = "image"
 	TarResourceType   = "tar"
 )
+
+const (
+	RoleBindingNameEditEnv = "zadig-env-edit"
+	RoleBindingNameReadEnv = "zadig-env-read"
+)
+
+var (
+	ClusterRoleEdit = &rbacv1beta1.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: RoleBindingNameEditEnv,
+		},
+		Rules: []rbacv1beta1.PolicyRule{rbacv1beta1.PolicyRule{
+			Verbs:     []string{"*"},
+			APIGroups: []string{""},
```

Comment:
`APIGroup = ""` means the core/v1 group, what about other api groups?

Reply:
change to "*"